### PR TITLE
[DDG-DA] Update crowd-sourced data results

### PR DESCRIPTION
**Pull request by wendili-cs**
[DDG-DA] Update crowd-sourced data results

## Description
Updating the experiment results on crowd-sourced data.

## Types of changes
- [ ] Fix bugs
- [ ] Add new feature
- [x] Update documentation

### CODE REVIEW

File: examples/benchmarks_dynamic/README.md [hunk 1]
```diff
@@ -4,15 +4,28 @@ So adapting the forecasting models/strategies to market dynamics is very importa
 
 The table below shows the performances of different solutions on different forecasting models.
 
-## Alpha158 dataset
+## Alpha158 Dataset
+- The label horizon of the `Alpha158` dataset is set to 20.
+- The rolling time intervals are set to 20 trading days.
+- The test rolling periods are from January 2017 to August 2020.
 
+### Internal Data
```
Comment (you-n-g) [inline examples/benchmarks_dynamic/README.md:12]: We can just remove this section.
It is not useful if the data is not public.

File: examples/benchmarks_dynamic/README.md [hunk 2]
```diff
@@ -4,15 +4,28 @@ So adapting the forecasting models/strategies to market dynamics is very importa
...
 | RR[Linear]       |Alpha158 |0.088|0.570|0.102    |0.622      |0.077              |1.175              |-0.086        |
 | DDG-DA[Linear]   |Alpha158 |0.093|0.622|0.106    |0.670      |0.085              |1.213              |-0.093        |
 | RR[LightGBM]     |Alpha158 |0.079|0.566|0.088    |0.592      |0.075              |1.226              |-0.096        |
 | DDG-DA[LightGBM] |Alpha158 |0.084|0.639|0.093    |0.664      |0.099              |1.442              |-0.071        |
 
-- The label horizon of the `Alpha158` dataset is set to 20.
-- The rolling time intervals are set to 20 trading days.
-- The test rolling periods are from January 2017 to August 2020.
+### Crowd Sourced Data
+Here is the [crowd sourced version of qlib data](data_collector/crowd_source/README.md): https://github.com/chenditc/investment_data/releases
```
Comment (you-n-g) [inline examples/benchmarks_dynamic/README.md:21]: Please add the reason why Qlib's public data is not used and this data-source is preferred.

File: examples/benchmarks_dynamic/README.md [hunk 3]
```diff
@@ -4,15 +4,28 @@ So adapting the forecasting models/strategies to market dynamics is very importa
...
 |------------------|---------|----|------|---------|-----------|-------------------|-------------------|--------------|
 | RR[Linear]       |Alpha158 |0.088|0.570|0.102    |0.622      |0.077              |1.175              |-0.086        |
 | DDG-DA[Linear]   |Alpha158 |0.093|0.622|0.106    |0.670      |0.085              |1.213              |-0.093        |
 | RR[LightGBM]     |Alpha158 |0.079|0.566|0.088    |0.592      |0.075              |1.226              |-0.096        |
 | DDG-DA[LightGBM] |Alpha158 |0.084|0.639|0.093    |0.664      |0.099              |1.442              |-0.071        |
 
-- The label horizon of the `Alpha158` dataset is set to 20.
-- The rolling time intervals are set to 20 trading days.
-- The test rolling periods are from January 2017 to August 2020.
+### Crowd Sourced Data
```
Comment (you-n-g) [inline examples/benchmarks_dynamic/README.md:20]: It will not necessary to use this name if we only release only results for one data source.

**Comment by you-n-g** [comment]
Please merge main branch to fix the CI.

The CI issue has been fixed in https://github.com/microsoft/qlib/pull/1398

File: examples/benchmarks_dynamic/README.md (hunk 2 above)
Comment (wendili-cs) [reply examples/benchmarks_dynamic/README.md:8]: The Yahoo version of qlib data does not contain `VWAP`, so all related factors are missing and filled with 0, which leads to a singular matrix and makes lower-level optimization of DDG-DA can not be solved.

**Pull request merged**